### DISCUSSION TITLE
Data engineering WG: Initial Package Reviews

### DISCUSSION THREAD

**Discussion by elimillera** [pharmaverse/pharmaverse#202] [General]
Mission Statement:
To coordinate and evaluate the efforts of open source developers working on data workflows and bridge the gaps between the needs of stakeholders and existing tools for data solutions.

Below I've listed the initial set of packages relating to data workflows

**Comment by elimillera** [comment]
- sdtmchecks
- roak
- datacutr
- admiral*
- xportr
- pkglite
- synthetic.cdisc.data

**Reply by elimillera** [reply]
@rossfarrugia This was my first pass of the pharmaverse packages. I see we have some overlap between the Tech/Templates working group. Do you have any strong feelings about the packages above?

**Reply by rossfarrugia** [reply]
@elimillera roak and datacutr are not yet ready (only listed as "Upcoming packages for consideration" for now) so they can be removed from the list and they'll come through as new package request issues for us to review later. the bottom 3 are under Tech/Templates WG currently but we could make a case to review them here instead under this WG - what would you prefer?

**Comment by elimillera** [comment]
- sdtmchecks
- admiral*
- xportr
- synthetic.cdisc.data

pkglite is being retained by the tech/templates team

**Comment by rossfarrugia** [comment]
@elimillera will close this discussion as all package reviews completed now by this WG
